nottrobin/blackgate-aggregator
# Markup and styling

I copied markup and styling from robinwinslow.co.uk, for expediency.

I slightly customised the styling the header to be black.
## QA

`docker-compose up` should start the sass watcher and build the sass.

Then visit localhost:8000 and see the new styling.
